### PR TITLE
CMakeLists: bump version to v9.1.0

### PR DESCRIPTION
**Pull request by benma**
- Added more ERC20 tokens

### CODE REVIEW

**Comment by x1ddos** [comment]
Well, backup tests use `v9.0.0` but I kind of assumed it's separate from the actual release version.

**Comment by x1ddos** [comment]
Maybe the quickest is to change https://github.com/digitalbitbox/bitbox02-firmware/blob/112a2378e8cb31bc7c94410eac7c1a1a4c0dba95/test/unit-test/test_restore.c#L97 and use `DIGITAL_BITBOX_VERSION_SHORT` instead of `"v9.0.0"`.

**Comment by benma** [comment]
> Maybe the quickest is to change
> 
> https://github.com/digitalbitbox/bitbox02-firmware/blob/112a2378e8cb31bc7c94410eac7c1a1a4c0dba95/test/unit-test/test_restore.c#L97
> and use `DIGITAL_BITBOX_VERSION_SHORT` instead of `"v9.0.0"`.

it was that before until a4aab382a2817a9919b7f937ecf7a4abba74bb8e, which you reviewed :smile: 

> backup: provide generator to backup_fill
> It is sort of an identifer for what created the backup. The way it was
before, the fixture test in test_backup.c will fail when we change the
firmware version. Proiding it explicitly means we can test with any
string in the test.

**Comment by x1ddos** [comment]
> it was that before until [a4aab38](https://github.com/digitalbitbox/bitbox02-firmware/commit/a4aab382a2817a9919b7f937ecf7a4abba74bb8e), which you reviewed smile
> 
> > backup: provide generator to backup_fill
> > It is sort of an identifer for what created the backup. The way it was
> > before, the fixture test in test_backup.c will fail when we change the
> > firmware version. Proiding it explicitly means we can test with any
> > string in the test.

Yeah, I missed it too, apparently.

> Proiding it explicitly means we can test with any string in the test.

I guess that didn't work. :)

**Comment by benma** [comment]
> I guess that didn't work. :)

seems it was only working for test_backup.c, where v9.0.0 is also hardcoded, against a fixture. 

fixed test_restore.c now

File: test/unit-test/test_restore.c [hunk 1]
```diff
@@ -388,21 +389,21 @@ int main(void)
...
-        cmocka_unit_test_setup_teardown(
-            test_restore_1_corrupt_backup_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_2_corrupt_backups_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_restore_3_corrupt_backups_from_directory, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(
-            test_fail_restore_if_majority_vote_fails, test_setup, test_teardown),
-        cmocka_unit_test_setup_teardown(test_restore_seed, test_setup, test_teardown),
+        /* cmocka_unit_test_setup_teardown( */
```
Comment (x1ddos) [inline test/unit-test/test_restore.c:392]: Wait, why are these commented out now, all except the first test?

Comment (benma) [reply test/unit-test/test_restore.c]: oh that's embarrassing :O fixed

Comment (x1ddos) [reply test/unit-test/test_restore.c]: Ah cool. For a second I thought they somehow weren't truly working anymore. :)

**Comment by x1ddos** [comment]
:+1:

**Pull request merged**